recipes/gstreamer/config.yml

```diff
@@ -1,13 +1,19 @@
 versions:
-  "1.16.2":
+  "1.20.5":
     folder: all
-  "1.18.0":
+  "1.20.4":
```

Comment:
Since we are adding a more recent from the same minor, I'd suggest dropping this one since it's unlikely to be used